src/common/grpc-expr/src/alter.rs

```diff
@@ -185,21 +192,24 @@ pub fn alter_expr_to_request(table_id: TableId, expr: AlterTableExpr) -> Result<
             },
             None => return MissingAlterIndexOptionSnafu.fail(),
         },
-        Kind::DropDefaults(o) => {
-            let names = o
-                .drop_defaults
-                .into_iter()
-                .map(|col| {
-                    ensure!(
-                        !col.column_name.is_empty(),
-                        MissingFieldSnafu {
-                            field: "column_name"
-                        }
-                    );
-                    Ok(col.column_name)
-                })
-                .collect::<Result<Vec<_>>>()?;
-            AlterKind::DropDefaults { names }
+        Kind::UnsetDefault(o) => AlterKind::UnsetDefault(o),
```

Comment:
Previously, in `DropDefaults`, we checked if the column name was empty. Do we still need this check after renaming it to `UnsetDefault`?